scripts/bootstrap-rootfs-overlay-hosted-server.sh

```diff
@@ -50,10 +55,20 @@ if [ -e ${output_dir} ]; then
      sudo chown -R $(id -u) ${output_dir}
      sudo chgrp -R $(id -g) ${output_dir}
 fi
+
+if [ "${region}" = "eu" ]; then
+    region_appendix="eu."
+elif [ "${region}" = "us" ]; then
```

Comment:
Should uppercase also be checked? I suppose it might be a pretty easy mistake to make.